app/subscriber/src/features/my-folders/MyFolders.tsx

```diff
@@ -34,13 +34,13 @@ export const MyFolders = () => {
     }).then((data) => {
       toast.success(`${data.name} created successfully`);
       setNewFolderName('');
-      setMyFolders([...myFolders, data]);
+      setMyFolders([data]);
```

Comment:
This will cause a similar issue.  When you add a folder it will remove all other folders that were originally in the array.